### PR TITLE
feat : Authentication 

### PR DESCRIPTION
**Pull request by muskan-gupta-0803**
- added login page
- added signup page

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -2,7 +2,8 @@
     --primary-color:black;
     --logo-color:#f2f2f2;
     --white-color:#fff;
-    --icon-badge:#0000ee;
+    --icon-badge:#3b82f6;
```
Comment (ajit-sharmaGH) [inline styles.css:5]: You can use css variables

**Review by ajit-sharmaGH — APPROVED** [review]

**Review by ajit-sharmaGH — APPROVED** [review]
code looks clean, good to merge

**Review by GeetaPogul — APPROVED** [review]
good to me

**Comment by athar05** [comment]
Looks good to merge

File: styles.css (hunk 1 above)
Comment (muskan-gupta-0803) [reply styles.css:5]: these are css variable declarations only.

**Comment by muskan-gupta-0803** [comment]
Thanks all for reviewing and approving.

**Pull request merged**